Backpack/src/main/res/values/attrs.xml

```diff
@@ -293,6 +293,31 @@
     <attr name="state_error" format="boolean" />
   </declare-styleable>
 
+  <declare-styleable name="BpkImageSkeleton">
+    <attr name="cornerType" format="enum">
+      <enum name="Square" value="0" />
```

Comment:
generally in xml the convention is to start with lowercase for enum values (this applies to all of these)

Reply:
Thanks for catch this, will update it.